Ginger/GingerCoreNETUnitTest/RunTestslib/CLITest.cs

```diff
@@ -102,8 +106,11 @@ public void OLDCLIConfigRegressionTest()
 
             // Act            
             CLIProcessor CLI = new CLIProcessor();
-            CLI.ExecuteArgs(new string[] { "ConfigFile=" + configFile });
-
+            
+            Task.Run(async () =>
+            {
+                await CLI.ExecuteArgs(new string[] { "ConfigFile=" + configFile });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)